doc/Installation/Install-LibreNMS.md

```diff
@@ -26,15 +26,15 @@ Connect to the server command line and follow the instructions below.
         apt install software-properties-common
         add-apt-repository universe
         apt update
-        apt install acl curl composer fping git graphviz imagemagick mariadb-client mariadb-server mtr-tiny nginx-full nmap php7.4-cli php7.4-curl php7.4-fpm php7.4-gd php7.4-json php7.4-mbstring php7.4-mysql php7.4-snmp php7.4-xml php7.4-zip rrdtool snmp snmpd whois unzip python3-pymysql python3-dotenv python3-redis python3-setuptools python3-systemd
+        apt install acl curl composer fping git graphviz imagemagick mariadb-client mariadb-server mtr-tiny nginx-full nmap php7.4-cli php7.4-curl php7.4-fpm php7.4-gd php7.4-gmp php7.4-json php7.4-mbstring php7.4-mysql php7.4-snmp php7.4-xml php7.4-zip rrdtool snmp snmpd whois unzip python3-pymysql python3-dotenv python3-redis python3-setuptools python3-systemd
```

Comment:
should we add it to composer.json? bit annoying since it halts if you dont have it.
could add it to suggested, but nobody read that anyway :)

Reply:
Suggested would be fine.  I don't want to make it a hard dependency yet.